content/docs/overview/about.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "About NuoDB Control Plane"
+description: ""
+summary: ""
+date: 2024-08-14T13:52:09+03:00
+lastmod: 2024-08-14T13:52:09+03:00
+draft: false
+weight: 15
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+NuoDB Control Plane (CP) enables customers to consume NuoDB in a Database as a Service (DBaaS) model.
```

Comment:
Suggested rework: 

The NuoDB Control Plane (CP) empowers customers to leverage NuoDB in a Database as a Service (DBaaS) model. Operating within a Kubernetes cluster, it manages the provisioning and administration of multiple administrative domains and their databases, all within the same cluster. NuoDB databases are dynamically created remotely using REST services, with a variety of predefined configuration options available.

Reply:
Thanks! It read better.